README.md

```diff
@@ -96,52 +90,33 @@ this.notify.alert('Unique Message', { id: 'some-unique-identifier' });
 
 ### Multiple Containers
 
-If you want to have separate notifications and control where they're inserted into the DOM you can have multiple `{{ember-notify}}` components, but only one of them can be accessed using the injected service.
+If you want to have separate notifications and control where they're inserted into the DOM you can have multiple `<EmberNotify />` components, but only one of them can be accessed using the injected service.
 The others you will need to provide a `source` property, so secondary containers should be used as follows:
 
-```hbs
-{{ember-notify source=someProperty}}
-```
-
-Using angle bracket invocation:
-
 ```hbs
 <EmberNotify @source={{someProperty}} />
```

Comment:
Haven't ever used that feature but if I got it right, this should be `{{this.alternativeNotify}}` to match the component JavaScript example, isn't it?

```suggestion
 <EmberNotify @source={{this.alternativeNotify}} />
```

Reply:
Yes, you got it right